From 767d48e0f60e7f858e8b31a88dd8cc1258e7ee9e Mon Sep 17 00:00:00 2001
From: Marcus Haehnel <marcus@mh-development.info>
Date: Sat, 21 Mar 2020 09:14:45 +0100
Subject: [PATCH] Fix invalid CMake syntax

The syntax of the foreach statement in data/kernels/CMakeLists.txt
was invalid. This lead to errors when running cmake.

Use correct syntax to make the build work.
---
 data/kernels/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/kernels/CMakeLists.txt b/data/kernels/CMakeLists.txt
index ce947bef9c..f605c18b7e 100644
--- a/data/kernels/CMakeLists.txt
+++ b/data/kernels/CMakeLists.txt
@@ -31,8 +31,8 @@ macro (testcompile_opencl_kernel IN)
 endmacro (testcompile_opencl_kernel)
 
 if (TESTBUILD_OPENCL_PROGRAMS)
-  foreach(IN ${DT_OPENCL_KERNELS})
-    testcompile_opencl_kernel(${IN})
+  foreach(KERNEL IN ITEMS ${DT_OPENCL_KERNELS})
+    testcompile_opencl_kernel(${KERNEL})
   endforeach()
 endif()
 
